Python/samples/cartpole/main.py

```diff
@@ -224,8 +253,11 @@ def test_random_policy(
         number of iterations to run, by default 10
     """
 
+    policy_name = "random" # TODO
```

Comment:
this is left as a TODO. would probably be hacky/weird to convert the `policy` object into a string so you can use it for the logger name, so should we just add an additional argument to the function for now?

Reply:
Good catch. Forgot about that.